SU-HSDO/stanford_mrc
# MRCD8-317 another one

# READY FOR REVIEW

# Summary
- Bolded the "Speaker" / "Speakers" labels and added a ":" to the end of them
- Placed speaker's name inline with label

# Needed By (Date)
- End of sprint (3/29)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Ensure that the speaker label is bolded and inline with the speaker's name

# Affected Projects or Products
- MRC
- stanford_mrc

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/MRCD8-317

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
